sylt-compiler/src/lua.rs

```diff
@@ -570,6 +571,43 @@ impl<'t> LuaCompiler<'t> {
                 write!(self, ";");
             }
 
+            Case { to_match, branches, fall_through } => {
+                // TODO(ed): This code cannot really be made better... Lua has no
+                // switch-statements.
+                write!(self, "local __case_tmp =");
```

Comment:
It becomes a variable if it's needed. But we need something so we can match the tag, that is this variables main purpous, since it's the tuple `{"TAG", value}`